docs/competency-api/audience.md

```diff
@@ -4,4 +4,46 @@ title: Audience
 nav_order: 2
 parent: Competency API
 ---
-# Audience
\ No newline at end of file
+# Audience
+## Update Audience
+
+### Definition
+```HTTP
+https://api.competency-constructor.clark.center/competencies/:competencyId/audience
+```
+
+## Headers
+```JSON
```

Comment:
From the screenshots, it seems the grey box surrounds the words but doesnt make a rectangle, like andy's screenshots. The only difference i could find was the he used `json` instead of `JSON` not sure if this actually make a difference. Could you try it out though?

Reply:
That does make a difference. I can't find any info as to why, but lowercase creates the box where uppercase highlights the words. @JosueMolinaMorales